resizing_hashtable/r_hashtables.py

```diff
@@ -42,7 +67,28 @@ def hash_table_insert(hash_table, key, value):
 # If you try to remove a value that isn't there, print a warning.
 # '''
 def hash_table_remove(hash_table, key):
-    pass
+    hashkey = hash(key, hash_table.capacity)
+    if hash_table.storage[hashkey] == None:
+        print("WARNING! Deleting a key that does not exist")
+        return None
+    else:
+        tmp = hash_table.storage[hashkey]
+        if tmp.next != None:
+            while tmp.next != None:
+                if tmp.next.key == key:
+                    tmp.next = tmp.next.next
+                    hash_table.count -= 1
+                    if hash_table.count >= 0.8 * hash_table.capacity:
+                        hash_table = hash_table_resize(hash_table)
+                    break
+                tmp = tmp.next
+        else:
+            if tmp.key == key:
+                hash_table.storage[hashkey] = None
```

Comment:
I don't think the hash_table.count is necessary to meet the spec but you'd want to update it here to keep it accurate